examples/SAP-to-Database/main.bal

```diff
@@ -0,0 +1,55 @@
+import ballerina/sql;
+import ballerinax/mysql;
+import ballerinax/mysql.driver as _;
+import ballerinax/sap.s4hana.api_masterinspcharacteristic_srv as mastinpchar;
+import ballerina/log;
+
+configurable S4HANAClientConfig s4hanaClientConfig = ?;
+configurable DatabaseConfig databaseConfig = ?;
+
+final mastinpchar:Client mastinpcharClient = check new (
+    config = {
+        auth: {
+            username: s4hanaClientConfig.username,
+            password: s4hanaClientConfig.password
+        }
+    },
+    hostname = s4hanaClientConfig.hostname
+);
+
+
+final mysql:Client companyDatabase = check new (
+    databaseConfig.host,
+    databaseConfig.username,
+    databaseConfig.password,
+    databaseConfig.databaseName,
+    databaseConfig.port
+);
+
+public function main() returns error? {
+    mastinpchar:CollectionOfA_InspectionSpecificationWrapper collectionOfInspectionSpecs = check mastinpcharClient->listA_InspectionSpecifications();
```

Comment:
This line exceeds 120 characters

Reply:
Improved here [Improve SAP to DB example](https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.qm/pull/23/commits/e6de654368da7d53a4bfc51ded0122365908f2b2)